pages/hardware/i2c-and-spi.md

```diff
@@ -76,16 +76,35 @@ An example of this is shown in our [Firebase Temperature Logger][firebaseTemp-li
 
 ### Raspberry Pi camera module
 
-In order to work with the Raspberry Pi camera module you will need to do the following:
+Depending on the version of your ResinOS, the system contains different version of the Raspberry Pi firmware, and you need to apply slightly different settings. In both cases you can either modify `config.txt` on the `resin-boot` partition of your SD card, or add the settings remotely by using `RESIN_HOST_CONFIG_variablename` settings in your [fleet or device configuration](/management/env-vars/).
```

Comment:
I see. I've fixed that in couple of places in the dashboard some time ago, but now found couple more. And it looks like what's shown for the new versions is actually in the **DB** and comes from the supervisor.